updateReadme.py

```diff
@@ -26,7 +26,7 @@ def main():
         "${fmtentries} | "
         "[link](http://sbc.io/hosts/${location}hosts)"
     )
-    with open(README_DATA_FILENAME, "r") as f:
+    with open(README_DATA_FILENAME, "r", encoding="utf-8") as f:
```

Comment:
This might be currently needed, but I guess it future proofs in case the json file includes any utf-8 characters in the future.

Reply:
Indeed, but `json.loads()` should handle that for us :-) But it's great to stay ahead of the future!